backend/app/app/api/deps.py

```diff
@@ -14,13 +15,26 @@
     tokenUrl=f"{settings.API_V1_STR}/login/access-token"
 )
 
-#Dependencies
+
+# Dependencies
 def get_db() -> Generator:
     try:
-        db = SessionLocal() #Only the code prior to and including the yield statement is executed before sending a response
-        yield db            #The yielded value is what is injected into path operations and other dependencies
+        db = (
+            SessionLocal()
+        )  # Only the code prior to and including the yield statement is executed before sending a response
+        yield db  # The yielded value is what is injected into path operations and other dependencies
+    finally:
+        db.close()  # The code following the yield statement is executed after the response has been delivered
+
+
+@contextmanager
+def db_context():
+    db = SessionLocal()
+    try:
+        yield db
     finally:
-        db.close()          #The code following the yield statement is executed after the response has been delivered
+        db.close()
```

Comment:
Just for curiosity can't you apply `@contextmanager` directly on the `get_db` or at least use `yield from` calling `get_db` to not repeat code.